app/models/game.rb

```diff
@@ -7,6 +7,23 @@ class Game < ApplicationRecord
   has_many :queens
   has_many :pieces
   scope :available, -> { where(black_player_id: nil) }
+  attr_accessor :turn_counter
```

Comment:
This may be where you looked at server side turn tracking

Reply:
Yes, remembering it now I think the server does do it. My real issue was trying to get JavaScript to read it as well. So I misremembered that this morning :) Sorry if that caused some confusion!

Reply:
Yeah, I made all the methods work with Game, but I was having a really hard time having JavaScript read it. There's likely a solution without JavaScript, but the pure JavaScript solution came to me faster. We can either look into implementing these methods or just delete them from Game.rb

Reply:
Since we have a functioning solution on the JavaScript side, let's remove these from the controller!